crates/cli/tests/snapshots/codegen__codegen_rust.snap

```diff
@@ -874,7 +855,11 @@ impl TableType for TestD {
 
 
 impl TestD {
-	}
+	#[allow(unused)]
+	pub fn filter_by_test_c(test_c: Option::<Namespace.testC>) -> TableIter<Self> {
```

Comment:
FWIW we decided not to allow filtering by optionals for now back in the consistent filtering proposal (https://github.com/clockworklabs/SpacetimeDBPrivate/pull/783).

So this method shouldn't have appeared.